frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -225,7 +225,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
       <Row style={{ marginTop: isDisabled ? 0 : 16 }}>
         <Col span={24}>
           {this.getEnableDisableLayerSwitch(isDisabled, onChange)}
-          <span style={{ fontWeight: 700 }}>{layerName}</span>
+          <span style={{ fontWeight: 700 }}>{isColorLayer ? layerName : "Volume Layer"}</span>
```

Comment:
It should only say "Volume Layer" if it's a volume (or hybrid) tracing :)

Reply:
Please check my newest changes. Is this what you meant? Now I only use the name "Volume Layer" if the layer is not a color layer and the tracing has a volume.

Reply:
Yes, looks good :+1: